### PR TITLE
Fix #3274 - explain HashAlgorithm parameter

### PR DESCRIPTION
**Pull request by sdwheeler**
Version(s) of document impacted
------------------------------
- [ ] Impacts 6.next document
- [x] Impacts 6 document
- [x] Impacts 5.1 document
- [x] Impacts 5.0 document
- [x] Impacts 4.0 document
- [x] Impacts 3.0 document

Reason(s) for not updating all version of documents
--------------------------------------------------
- [ ] The documented feature was introduced in version (list version here) of PowerShell
- [ ] This issue only shows up in version (list version(s) here) of the document
- [ ] This PR partially fixes the issue, and issue #<insert here> tracks the remaining work

### CODE REVIEW

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[236ee6b](https://github.com/sdwheeler/PowerShell-Docs/pull/3739/commits/236ee6ba83807ac418f0f85ebc28b210ca6897d2)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 9 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[236ee6b](https://github.com/sdwheeler/PowerShell-Docs/pull/3739/commits/236ee6ba83807ac418f0f85ebc28b210ca6897d2)_: 

### :x: Validation status: errors

Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master#build--publish) which may help to resolve issue.


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[]() | :x:Error | | [Details](#user-content-)

<a name=""></a>
### []()
 - **[Error]** Cannot sync git repo to specified state. Please make sure the pull request is mergeable. Details: Git merge results in conflict, repository remote url: https://github.com/MicrosoftDocs/PowerShell-Docs, working directory: W:\w5gu-s, target branch: staging, target branch commit: 8710e32a920e8e174f9be52133b2dc9a59169962, source branch: 6809f03c7826ebcfb443fe6e60fe46e6/sdw-issue3274, source branch commit: .

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C20%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201902201908033074-3739%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=9li8XndHO9KSRR7IdlqfizErvasM6sWzPUMHDfmmGzY%3D&st=2019-02-20T19%3A03%3A39Z&se=2019-03-23T19%3A08%3A39Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: reference/3.0/Microsoft.PowerShell.Security/Set-AuthenticodeSignature.md [hunk 1]
```diff
@@ -10,99 +10,109 @@ title:  Set-AuthenticodeSignature
 # Set-AuthenticodeSignature
 
 ## SYNOPSIS
-
-Adds an Authenticode signature to a Windows PowerShell script or other file.
+Adds an Authenticode signature to a PowerShell script or other file.
```
Comment (DCtheGeek) [inline reference/3.0/Microsoft.PowerShell.Security/Set-AuthenticodeSignature.md]: I'd provide a link out to the overview for an Authenticode signature: https://docs.microsoft.com/en-us/windows-hardware/drivers/install/authenticode

File: reference/3.0/Microsoft.PowerShell.Security/Set-AuthenticodeSignature.md [hunk 2]
```diff
@@ -10,99 +10,109 @@ title:  Set-AuthenticodeSignature
...
+the drive exposed by the certificate provider. The **CodeSigningCert** parameter, which is supported
+only by the certificate provider, limits the certificates retrieved to those with code-signing
+authority. The command stores the result in the `$cert` variable.
 
-The first command uses the Get-ChildItem cmdlet and the Windows PowerShell certificate provider to get the certificates in the Cert:\CurrentUser\My subdirectory of the certificate store.
-(The Cert: drive is the drive exposed by the certificate provider.) The CodeSigningCert parameter, which is supported only by the certificate provider, limits the certificates retrieved to those with code-signing authority.
-The command stores the result in the $cert variable.
+The second command uses the `Set-AuthenticodeSignature` cmdlet to sign the `PSTestInternet2.ps1`
+script. It uses the **FilePath** parameter to specify the name of the script and the **Certificate**
+parameter to specify that the certificate is stored in the $cert variable.
```
Comment (DCtheGeek) [inline reference/3.0/Microsoft.PowerShell.Security/Set-AuthenticodeSignature.md]: $cert should be codefenced.

File: reference/3.0/Microsoft.PowerShell.Security/Set-AuthenticodeSignature.md [hunk 3]
```diff
@@ -145,9 +155,12 @@ Accept wildcard characters: False
...
 
-For Windows PowerShell 3.0, the default is SHA256, which is the Windows default hashing algorithm.
-For Windows PowerShell 2.0, the default is SHA1.
-Files that are signed with a different hashing algorithm might not be recognized on other systems.
+For PowerShell 3.0, the default is SHA256, which is the Windows default hashing algorithm. For
+PowerShell 2.0, the default is SHA1. Files that are signed with a different hashing algorithm might
+not be recognized on other systems. Which algorithms are supported depends on the version of the
+operating system.
+
+For a list of possible values, see [HashAlgorithmName Struct ](/dotnet/api/system.security.cryptography.hashalgorithmname?view=netframework-4.7.2#properties).
```
Comment (DCtheGeek) [inline reference/3.0/Microsoft.PowerShell.Security/Set-AuthenticodeSignature.md]: Given the framework support at that link, are all the options supported in PS3?

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[61951e0](https://github.com/sdwheeler/PowerShell-Docs/pull/3739/commits/61951e0e25c25d5a86a5c61915d54aae93d81253)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 9 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[61951e0](https://github.com/sdwheeler/PowerShell-Docs/pull/3739/commits/61951e0e25c25d5a86a5c61915d54aae93d81253)_: 

:clock10: **Incremental building**: average incremental building time is _12 min(s) 49 sec(s)_

File: reference/3.0/Microsoft.PowerShell.Security/Set-AuthenticodeSignature.md (hunk 1 above)
Comment (sdwheeler) [reply reference/3.0/Microsoft.PowerShell.Security/Set-AuthenticodeSignature.md]: Added

File: reference/3.0/Microsoft.PowerShell.Security/Set-AuthenticodeSignature.md (hunk 2 above)
Comment (sdwheeler) [reply reference/3.0/Microsoft.PowerShell.Security/Set-AuthenticodeSignature.md]: Fixed

File: reference/3.0/Microsoft.PowerShell.Security/Set-AuthenticodeSignature.md (hunk 3 above)
Comment (sdwheeler) [reply reference/3.0/Microsoft.PowerShell.Security/Set-AuthenticodeSignature.md]: Yeah, its the same list.

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[61951e0](https://github.com/sdwheeler/PowerShell-Docs/pull/3739/commits/61951e0e25c25d5a86a5c61915d54aae93d81253)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/3.0/Microsoft.PowerShell.Security/Set-AuthenticodeSignature.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3274/reference/3.0/Microsoft.PowerShell.Security/Set-AuthenticodeSignature.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.security/set-authenticodesignature?view=powershell-3.0&branch=pr-en-us-3739) (powershell-3.0) |
[reference/4.0/Microsoft.PowerShell.Security/Set-AuthenticodeSignature.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3274/reference/4.0/Microsoft.PowerShell.Security/Set-AuthenticodeSignature.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.security/set-authenticodesignature?view=powershell-4.0&branch=pr-en-us-3739) (powershell-4.0) |
[reference/5.0/Microsoft.PowerShell.Security/Set-AuthenticodeSignature.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3274/reference/5.0/Microsoft.PowerShell.Security/Set-AuthenticodeSignature.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.security/set-authenticodesignature?view=powershell-5.0&branch=pr-en-us-3739) (powershell-5.0) |
[reference/5.1/Microsoft.PowerShell.Security/Set-AuthenticodeSignature.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3274/reference/5.1/Microsoft.PowerShell.Security/Set-AuthenticodeSignature.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.security/set-authenticodesignature?view=powershell-5.1&branch=pr-en-us-3739) (powershell-5.1) |
[reference/6/Microsoft.PowerShell.Security/Set-AuthenticodeSignature.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3274/reference/6/Microsoft.PowerShell.Security/Set-AuthenticodeSignature.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.security/set-authenticodesignature?view=powershell-6&branch=pr-en-us-3739) (powershell-6) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C20%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201902201923174589-3739%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=JPsuvy76WYYmhc2eQlOWpnZFhrGCI7EbUbEsWpJWxoY%3D&st=2019-02-20T19%3A32%3A01Z&se=2019-03-23T19%3A37%3A01Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[9c7b552](https://github.com/sdwheeler/PowerShell-Docs/pull/3739/commits/9c7b552fbb6dad260d74c850022d0e94e4ebab8c)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 9 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[9c7b552](https://github.com/sdwheeler/PowerShell-Docs/pull/3739/commits/9c7b552fbb6dad260d74c850022d0e94e4ebab8c)_: 

:clock10: **Incremental building**: average incremental building time is _12 min(s) 49 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[9c7b552](https://github.com/sdwheeler/PowerShell-Docs/pull/3739/commits/9c7b552fbb6dad260d74c850022d0e94e4ebab8c)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/3.0/Microsoft.PowerShell.Security/Set-AuthenticodeSignature.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3274/reference/3.0/Microsoft.PowerShell.Security/Set-AuthenticodeSignature.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.security/set-authenticodesignature?view=powershell-3.0&branch=pr-en-us-3739) (powershell-3.0) |
[reference/4.0/Microsoft.PowerShell.Security/Set-AuthenticodeSignature.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3274/reference/4.0/Microsoft.PowerShell.Security/Set-AuthenticodeSignature.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.security/set-authenticodesignature?view=powershell-4.0&branch=pr-en-us-3739) (powershell-4.0) |
[reference/5.0/Microsoft.PowerShell.Security/Set-AuthenticodeSignature.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3274/reference/5.0/Microsoft.PowerShell.Security/Set-AuthenticodeSignature.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.security/set-authenticodesignature?view=powershell-5.0&branch=pr-en-us-3739) (powershell-5.0) |
[reference/5.1/Microsoft.PowerShell.Security/Set-AuthenticodeSignature.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3274/reference/5.1/Microsoft.PowerShell.Security/Set-AuthenticodeSignature.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.security/set-authenticodesignature?view=powershell-5.1&branch=pr-en-us-3739) (powershell-5.1) |
[reference/6/Microsoft.PowerShell.Security/Set-AuthenticodeSignature.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3274/reference/6/Microsoft.PowerShell.Security/Set-AuthenticodeSignature.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.security/set-authenticodesignature?view=powershell-6&branch=pr-en-us-3739) (powershell-6) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C20%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201902201932069952-3739%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=1iA7ia7e8Yu9kw0l10L61QPxELmKmE6lhQOwTl%2Bybf8%3D&st=2019-02-20T19%3A46%3A00Z&se=2019-03-23T19%3A51%3A00Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by DCtheGeek** [comment]
Fixed #3724

**Pull request merged**